From fac82d9ddb4d0bddfbe5a97d23f832f329ffed79 Mon Sep 17 00:00:00 2001
From: Takeshi Kihara <takeshi.kihara.df@renesas.com>
Date: Mon, 12 Mar 2018 21:00:48 +0900
Subject: [PATCH 123/909] clk: renesas: r8a7796: Add division value of zg clock

commit b4d4b81ee84a742eb8881b877c396c5afe6406ac from
git://git.kernel.org/pub/scm/linux/kernel/git/horms/renesas-bsp.git

This patch set the division value of the zg clock divider for the R8A7796
SoC.

Signed-off-by: Takeshi Kihara <takeshi.kihara.df@renesas.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/clk/renesas/r8a7796-cpg-mssr.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/clk/renesas/r8a7796-cpg-mssr.c b/drivers/clk/renesas/r8a7796-cpg-mssr.c
index 24d618aca21e..0543e408a40f 100644
--- a/drivers/clk/renesas/r8a7796-cpg-mssr.c
+++ b/drivers/clk/renesas/r8a7796-cpg-mssr.c
@@ -79,7 +79,7 @@ static const struct cpg_core_clk r8a7796_core_clks[] __initconst = {
 	/* Core Clock Outputs */
 	DEF_GEN3_Z("z",         R8A7796_CLK_Z,  CLK_TYPE_GEN3_Z,  CLK_PLL0, 2),
 	DEF_GEN3_Z("z2",        R8A7796_CLK_Z2, CLK_TYPE_GEN3_Z2, CLK_PLL2, 2),
-	DEF_BASE("zg",          R8A7796_CLK_ZG,    CLK_TYPE_GEN3_ZG, CLK_PLL4),
+	DEF_GEN3_Z("zg",        R8A7796_CLK_ZG, CLK_TYPE_GEN3_ZG, CLK_PLL4, 4),
 	DEF_FIXED("ztr",        R8A7796_CLK_ZTR,   CLK_PLL1_DIV2,  6, 1),
 	DEF_FIXED("ztrd2",      R8A7796_CLK_ZTRD2, CLK_PLL1_DIV2, 12, 1),
 	DEF_FIXED("zt",         R8A7796_CLK_ZT,    CLK_PLL1_DIV2,  4, 1),
-- 
2.17.1

